migration/pom.xml

```diff
@@ -0,0 +1,38 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <parent>
+        <groupId>org.axonframework</groupId>
+        <artifactId>axon</artifactId>
+        <version>4.7.2-SNAPSHOT</version>
+    </parent>
+
+    <artifactId>migration</artifactId>
+
+    <name>Axon Framework - Migration</name>
+    <description>
+        Module containing the migration recipes.
+    </description>
+
+    <dependencies>
+        <dependency>
+            <groupId>org.axonframework</groupId>
```

Comment:
I guess you're only depending on `axon-messaging`, as that's the only migration you test, right?

Reply:
If there's moved classes in other modules they would have to be included here as well to ensure we can resolve the types *after* the first round of migration. That way any subsequent recipes can be sure to have complete type information. I thought most affected classes were in axon-messaging, but feel free to include other modules that you know are affected. Like I said this was a quick outline before work. :)

Reply:
Thanks for confirming my assumption, @timtebeek!